src/models/JsonPatch.ts

```diff
@@ -0,0 +1,32 @@
+export enum Ops {
+	replace = 'replace',
+	remove = 'remove',
```

Comment:
May be best to keep naming and values consistent with these `enums` (capitalised names, upper-case values)

Reply:
Agree on the capitalised names, however the value itself should be lowercase (JSONPatch doesn't specifically say what case, but the docs all use lower)